docs/getting-started/setup/macos.md

```diff
@@ -196,9 +240,17 @@ export NVM_DIR="$HOME/.nvm"
 brew install yarn
 ```
 
+You should now have `yarn` available on your command line. Check the version with:
+
+```shell
+yarn --version
+```
+
+You should be using Yarn 1.22.19 or later.
+
 # Install Podman
 
-This project uses [Podman](https://podman.io/) for containerization.
+This project uses [Podman](https://podman.io/) for containerization. Podman is a daemonless container engine for developing, managing, and running Open Container Initiative containers. It is an alternative to Docker and is available without fee for macOS via Homebrew.
```

Comment:
I had to install a specific version of podman (4.9.3) that I could not get with Homebrew.
I downloaded the file podman-installer-macos-arm64.pkg from https://github.com/containers/podman/releases/tag/v4.9.3.

Reply:
Good to know for reference. If you're able to isolate the issue you were having with Podman 5.x perhaps that's worth a support ticket to the Podman repo (unrelated [example issue](https://github.com/containers/podman/issues/22462) I opened yesterday).